cellprofiler/modules/measureimageskeleton.py

```diff
@@ -0,0 +1,156 @@
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import scipy.ndimage
+import skimage.measure
+import skimage.segmentation
+import skimage.util
+
+
+def _neighbors(image):
```

Comment:
Can you please add a comment explaining this procedure? A link to an article would be fine, too.